frontend/app/.server/domain/multi-channel/search-api-service.ts

```diff
@@ -0,0 +1,12 @@
+import type { HitListResult } from '~/.server/domain/multi-channel/search-api-models';
+import { getDefaultSearchService } from '~/.server/domain/multi-channel/search-api-service-default';
+import { getMockSinSearchService } from '~/.server/domain/multi-channel/search-api-service-mock';
+import { serverEnvironment } from '~/.server/environment';
+
+export type SinSearchService = {
+  getSearchResults(caseId: string): Promise<HitListResult[] | undefined>;
```

Comment:
I don't think this should ever return `undefined`. If no search results are found, return `[]`.